lib/contracts/formatters.rb

```diff
@@ -44,6 +44,7 @@ def initialize(value)
       # from standard Strings.
       # Primitive values e.g. 42, true, nil will be left alone.
       def inspect
+        return @value.inspect if empty_val?
```

Comment:
I'm not sure this is needed. Please add a test case showing that the formatting would look wrong without it, see other tests I added.